state/migration_export.go

```diff
@@ -1807,7 +1807,7 @@ func (e *exporter) addVolume(vol *volume, volAttachments []volumeAttachmentDoc)
 		va := volumeAttachment{doc}
 		logger.Debugf("  attachment %#v", doc)
 		args := description.VolumeAttachmentArgs{
-			Machine: va.Machine(),
+			Machine: va.Host().(names.MachineTag),
```

Comment:
Is this just because export isn't supported for caas models yet? Shouldn't description be updated to use Host here too?

Reply:
It will need to be updated at some point, but we need to rev the db description format etc. For  now, it's ok to leave as machine since we don't support migration yet